src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -634,6 +667,128 @@ def get_request_status(
     return paginated
 
 
+@router.get(
+    PRIVACY_REQUESTS,
+    dependencies=[Security(verify_oauth_client, scopes=[PRIVACY_REQUEST_READ])],
+    response_model=Page[
+        Union[
+            PrivacyRequestVerboseResponse,
+            PrivacyRequestResponse,
+        ]
+    ],
+    deprecated=True,
```

Comment:
Flagging as `deprecated` so it's reflected in Swagger
<img width="1414" alt="image" src="https://github.com/ethyca/fides/assets/2092272/051fbfab-341f-4e22-a8e5-e4e0d3c63f7b">